deeprank2/molstruct/structure.py

```diff
@@ -27,26 +32,24 @@ def __repr__(self) -> str:
     def has_chain(self, chain_id: str) -> bool:
         return chain_id in self._chains
 
-    def get_chain(self, chain_id: str):
+    def get_chain(self, chain_id: str) -> "Chain":
```

Comment:
Why Chain is not under the `if TYPE_CHECKING:`?

Reply:
because it comes from the same module, so doesn't need to be imported (in fact, can't be). It still needs the quotation marks, because it is only defined afterwards.

Reply:
oh sorry I didn't notice that it was defined there